apps/style/applab/skins/base.scss

```diff
@@ -39,7 +39,7 @@
     font-size: 14px;
   }
 
-  input, div[contenteditable=true], button, select, img, canvas, label {
+  input, div[contenteditable=true], button, select, img, label {
```

Comment:
The margin on canvas meant that if I created a canvas at 0, 0 of 320x480, there would still be a 5px gap on the left/top. Notice that there's still some weirdness in that the design mode representation has a border that gets cut off on the right/bottom 
![image](https://cloud.githubusercontent.com/assets/1767466/8336346/a2ec2628-1a58-11e5-9ac1-f1a104f7e3d3.png)